experimental/uefi/.cargo/config.toml

```diff
@@ -1,9 +1,14 @@
 [build]
 target = "x86_64-unknown-uefi"
 
-[target.x86_64-unknown-uefi]
+# Test runners get only one serial port, routed to stdio.
+[target.'cfg(test)']
 runner = "qemu-system-x86_64 -nodefaults -nographic -bios /usr/share/OVMF/OVMF_CODE.fd -serial stdio -machine q35 -device isa-debug-exit,iobase=0xf4,iosize=0x04 -kernel"
 
+# Otherwise, (a) the first serial port gets routed to a log, and (b) the second serial gets attached to stdio.
```

Comment:
I assume this is temporary, and the second serial port will be routed to a UDS that is used by the untrusted loader for communication.

Reply:
Yep, the untrusted loader will need to set up sockets for these serial ports.